handbook/engineering/onboarding/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,25 @@
+# Engineering manager onboarding
+
+We expect our engineering managers to drive outcomes, so here are the outcomes we expect of you as you are onboarding at Sourcegraph. You have ownership and autonomy on how to achieve these outcomes. If you aren't confident that you are on the right path, just ask for help from your team and/or manager!
+
+## Week 1
+
+- You have completed [general setup](index.md#getting-set-up).
```

Comment:
```suggestion
- You have completed the [engineering setup](index.md#getting-set-up).
```

This otherwise might be mixed up with `../../people-ops/onboarding/index.md#general-onboarding-checklist`